opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/UtilsTest.java

```diff
@@ -102,19 +102,20 @@ public void testGetDueProfileImageResourceIDentifier() {
-
-        Event event = gson.fromJson(oldIllnessInfo, Event.class);
-        EventClient eventClient = new EventClient(event,new Client("12345"));
-        List<EventClient> result = Utils.processOldEvents(eventClient);
-        Assert.assertEquals(result.size(),1);
+    @Test
+    public void testingConvertDpToPixel() {
+        Assert.assertEquals(0.0, Utils.convertDpToPixel(0, context), 0);
+    }
+
+    @Test
+    public void testRandomGeneratedId() {
+        Assert.assertNotEquals(2, Utils.getRandomGeneratedId());
+    }
+
```

Comment:
Will the function always not return 2 as the random generated id?

Reply:
Its removed